tests/unit/test_simulations.py

```diff
@@ -78,3 +79,63 @@ def test_all_GKP_high_squeezing(self, code):
         errors_py = ec_monte_carlo(code, trials, delta, p_swap, passive_objects=passive_objects)
         # Check that there are no errors in all-GKP high-squeezing limit.
         assert errors_py == 0
+
+
+@pytest.mark.parametrize("passive", [True, False])
+@pytest.mark.parametrize("empty_file", [True, False])
+def test_simulation_output_file(tmpdir, passive, empty_file):
+    """Check the content of the simulation output file."""
+
+    f = tmpdir.join("sims_results.csv")
+    if not empty_file:
+        f.write_text(
+            "distance,ec,boundaries,delta,p_swap,errors_py,trials,current_time\n"
+            + "2,primal,open,0.04,0.5,1,10,10:06:01\n",
+            encoding="UTF-8",
+        )
```

Comment:
What is the purpose of this?

Reply:
To test that the simulation is appending results to an existing file. This lines creates a dummy file before executing the simulation.